net/gensio/Config.in

```diff
@@ -0,0 +1,39 @@
+	config GENSIO_SSL
+		depends on PACKAGE_libgensio
+		bool "Enable SSL-support via OpenSSL"
+		default n
+
+	config GENSIO_SSHD
+		depends on GENSIO_SSL
+		bool "Build gtlsshd"
```

Comment:
This should not be configurable in the menu of the library IMO. I'd prefer to have this as option of gensio-bin and then it should select GENSIO_SSL and not depend on it.

Reply:
Agreed, changed.